docs/configuration.rst

```diff
@@ -160,9 +160,9 @@ These settings are global and have effect on both, tooltips and modal dialogues.
 
 .. confval:: hoverxref_api_host
 
-   Description: Host URL for the API to retrieve the content of the floating window
+   Description: Host or root URL for the API to retrieve the content of the floating window
 
-   Default: ``https://readthedocs.org``
+   Default: ``/_``
```

Comment:
I think we only want to default this value when running on RTD, don't we? For dev setup this will cause weird breakage for a lot of users.

Reply:
It's documented that this should be changed to rtd.org in the dev docs, for the docs of this extension we are checking if we are running on rtd

https://github.com/readthedocs/sphinx-hoverxref/blob/f6e533f74cad701e436eaa1cc3fbff374f6a942c/docs/conf.py#L66-L68

People testing this locally will need to define the `hoverxref_project` and `hoverxref_version` anyway.